File: 2148/2148.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var countElementsWithStrictlySmall = function(root) {
    if (!root) return 0;
    return 1 + Math.max(countElementsWithStrictlySmall(root.left), countElementsWithStrictlySmall(root.right));
};

File: 2148/2148.cs
public class Solution {
    public int CountElementsWithStrictlySmall(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(CountElementsWithStrictlySmall(root.left), CountElementsWithStrictlySmall(root.right));
    }
}